.travis.yml

```diff
@@ -49,6 +51,8 @@ matrix:
     - compiler: clang
       env: PRESET=CC3200 XI_MAKETARGET=""
```

Comment:
Sorry for being fussy but I think it would be good to add postfix _SDK110 to this CC3200 in order to stay informative and consistent with naming.